Makefile

```diff
@@ -349,6 +348,7 @@ $(OUTPUT_DIR)/btfhub:
 LIBBPF_CFLAGS = "-fPIC"
 LIBBPF_LDFLAGS =
 LIBBPF_SRC = ./3rdparty/libbpf/src
+LIBBPF_INCLUDE = $(abspath ./3rdparty/libbpf/include/uapi)
```

Comment:
At line 401 we have `-I ./3rdparty/include \` for `$(OUTPUT_DIR)/tracee.bpf.o` target. I ask you, how about to also change it to use the variable?

A doubt: the full uapi path should be used? wouldn't only `-I ./3rdparty/include` work?

Reply:
We need to include full `./3rdparty/libbpf/include/uapi` in order to work. So I'll rename to `LIBBPF_INCLUDE_UAPI` as discussed offline, tks for the tip.